test/feature/testutil/docker_composer.go

```diff
@@ -138,7 +138,7 @@ func (dc *DockerComposer) Up(env []string) error {
 
 // Down trears all containers/VMs dowwn
 func (dc *DockerComposer) Down() error {
-	return dc.runCompose([]string{"down", "-v", "-t", strconv.Itoa(int(defaultDockerComposeTimeout / time.Second))}, nil)
+	return dc.runCompose([]string{"down", "-v" /* "-t", strconv.Itoa(int(defaultDockerComposeTimeout / time.Second))*/}, nil)
```

Comment:
?

Reply:
This part is significantly slowing tests. Single basic test runs 9 min with this line